server/Services/IOrderService.cs

```diff
@@ -0,0 +1,15 @@
+﻿using server.DTO;
+using server.Models.DB;
+
+namespace server.Services
+{
+    public interface IOrderService
+    {
+        Task<List<OrderReturnDto>> AllOrdersServiceAsync();
```

Comment:
This can be written as GetAllOrdersServicAsync() to improve readability . 
Please update the method names accordingly.

Reply:
Good one, resolved